SCHEMA.sql

```diff
@@ -9,9 +9,9 @@ CREATE TABLE IF NOT EXISTS products (
   imageUrl VARCHAR(120),
   category INT,
   isFavorite INT,
-  price DECIMAL(4,2),
-  cutPrice DECIMAL(4,2),
```

Comment:
Is the database not handling decimals correctly? Why change it from a decimal to varchar?